src/libs/ck-libs/channel/channel.C

```diff
@@ -0,0 +1,3 @@
+#include "charm++.h"
```

Comment:
Since the code is heavily templated, most code needs to be in the header at present. `*.C` was included to future-proof/foster possible additions of non-templated code in the correct location. That being said, it can safely be removed from now.